integration_test/test/signin_test.dart

```diff
@@ -0,0 +1,102 @@
+import 'dart:async';
+import 'package:flutter/material.dart';
+import 'package:flutter_template/core/di/di_provider.dart';
+import 'package:flutter_template/core/model/authentication_status.dart';
+import 'package:flutter_template/main.dart' as app;
+import 'package:flutter_template/ui/extensions/context_extensions.dart';
+import 'package:flutter_template/ui/router/app_router.dart';
+import 'package:flutter_template/ui/section/error_handler/global_event_handler_cubit.dart';
+import 'package:flutter_template/ui/signin/signin_screen.dart';
+import 'package:flutter_template/ui/welcome/welcome_screen.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:rxdart/rxdart.dart';
+
+import '../common/general_helpers.dart';
+import '../common/repository_mocks.dart';
```

Comment:
check this out

Reply:
These files are in the test folder and outside of lib, so the unique way to import them is using a relative path.